Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -937,12 +937,13 @@ RefPtr<StyleRuleFontPaletteValues> CSSParserImpl::consumeFontPaletteValuesRule(C
     Vector<FontPaletteValues::OverriddenColor> overrideColors;
     if (auto overrideColorsValue = properties->getPropertyCSSValue(CSSPropertyOverrideColors)) {
         const auto& list = downcast<CSSValueList>(*overrideColorsValue);
-        for (const auto& item : list) {
-            const auto& pair = downcast<CSSFontPaletteValuesOverrideColorsValue>(item);
-            if (!pair.key().isInteger())
+        for (Ref item : list) {
+            Ref pair = downcast<CSSFontPaletteValuesOverrideColorsValue>(item);
+            Ref pairKey = pair->key();
+            if (!pairKey->isInteger())
                 continue;
-            auto key = pair.key().resolveAsIntegerDeprecated<unsigned>();
-            auto color = CSSColorValue::absoluteColor(pair.color());
+            auto key = pairKey->resolveAsIntegerDeprecated<unsigned>();
+            auto color = CSSColorValue::absoluteColor(Ref { pair->color() });
```

Comment:
If we mark `CSSFontPaletteValuesOverrideColorsValue::m_color` as `const`, then I believe we won't need to `Ref { }` here.